filebeat/fileset/modules_test.go

```diff
@@ -397,16 +397,6 @@ func TestInterpretError(t *testing.T) {
 		Input  string
 		Output string
 	}{
-		{
```

Comment:
Do we need some new tests for the new functionality?

Reply:
@ruflin Hmm since now these two plugins are modules which will always be there. Missing other plugins will still be captured by `interpretError` function and tested by `"other plugin not installed"` in `TestInterpretError`. I can't think of anything to add here. Do you have any test cases in mind?

Reply:
Seeing that we have below `other plugin not installed` in think we are good here.